### PR TITLE
Fix role update issue #46

### PR DESCRIPTION
**Pull request by dreamsoftech**
From #46:

>When role titles are camel cased, it creates new role named downcased instead of updating properly.
>For example, when I have role named `Community Member`, it creates new role named `communitymember` instead of updating role by `community_member`.

### CODE REVIEW

**Comment by parndt** [comment]
Hello, thanks for this, but what is the issue it's fixing? There's no description for your PR.

File: app/controllers/refinery/authentication/devise/admin/users_controller.rb [hunk 1]
```diff
@@ -47,7 +47,7 @@ def update
 
             store_user_memento
 
-            @user.roles = @selected_role_names.map { |r| Refinery::Authentication::Devise::Role[r.downcase] }
+            @user.roles = @selected_role_names.map { |r| Refinery::Authentication::Devise::Role[r.underscore] }
```
Comment (parndt) [inline app/controllers/refinery/authentication/devise/admin/users_controller.rb]: @dreamsoftech down on lines 66 and 122 it's still using `Refinery::Authentication::Devise::Role[r.downcase]` and `@selected_role_names.map(&:downcase)`

Comment (dreamsoftech) [reply app/controllers/refinery/authentication/devise/admin/users_controller.rb]: @parndt Good Catch!

File: app/controllers/refinery/authentication/devise/admin/users_controller.rb [hunk 2]
```diff
@@ -63,7 +63,7 @@ def create_successful
             # if the user is a superuser and can assign roles according to this site's
             # settings then the roles are set with the POST data.
             if user_can_assign_roles?
-              @user.roles = @selected_role_names.map { |r| Refinery::Authentication::Devise::Role[r.downcase] }
+              @user.roles = @selected_role_names.map(&:underscore)
```
Comment (parndt) [inline app/controllers/refinery/authentication/devise/admin/users_controller.rb]: this was supposed to turn the array into roles

```suggestion
              @user.roles = @selected_role_names.map { |r| Refinery::Authentication::Devise::Role[r.underscore] }
```

File: app/controllers/refinery/authentication/devise/admin/users_controller.rb [hunk 3]
```diff
@@ -37,17 +37,17 @@ def edit
...
-              flash.now[:error] = t('lockout_prevented', :scope => 'refinery.authentication.devise.admin.users.update')
-              render :edit and return
+              flash.now[:error] = t('lockout_prevented', scope: 'refinery.authentication.devise.admin.users.update')
+              render :edit && return
             end
 
             store_user_memento
 
-            @user.roles = @selected_role_names.map { |r| Refinery::Authentication::Devise::Role[r.downcase] }
+            @user.roles = @selected_role_names.map(&:underscore)
```
Comment (parndt) [inline app/controllers/refinery/authentication/devise/admin/users_controller.rb]: this was supposed to turn the array into roles

```suggestion
              @user.roles = @selected_role_names.map { |r| Refinery::Authentication::Devise::Role[r.underscore] }
```

**Comment by parndt** [comment]
@dreamsoftech ^ I reviewed this for you 😄

**Comment by dreamsoftech** [comment]
@parndt Pushed changes

**Review by parndt — DISMISSED** [review]
@bricesanchez looks okay to me; what do you think?

**Comment by parndt** [comment]
For some reason, CI is seriously unhappy with it.

**Comment by dreamsoftech** [comment]
@parndt It seems like it needs to rebase this branch against master.

**Review by bricesanchez — DISMISSED** [review]
Thanks for this bugfix! 
I will be ok to merge it if CI goes green

**Comment by parndt** [comment]
thank you!

**Pull request merged**